prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -110,12 +110,11 @@ items:
                       fieldPath: spec.nodeName
               image: 'weaveworks/weave-kube:latest'
               imagePullPolicy: Always
-              livenessProbe:
+              readinessProbe:
                 httpGet:
                   host: 127.0.0.1
                   path: /status
                   port: 6784
-                initialDelaySeconds: 30
```

Comment:
I'd still keep this. There's no need for the Kubelet to check for readiness immediately, taking pressure off that path?

Reply:
On second thoughts: "meh".  It's a small code path and the readiness probe fires every 10 seconds, so hardly any load.